src/binutil/server.rs

```diff
@@ -346,7 +350,7 @@ fn run_raft_server(pd_client: RpcClient, cfg: &TiKvConfig, security_mgr: Arc<Sec
     let mut status_server = StatusServer::new(server_cfg.status_thread_pool_size);
     if status_enabled {
         // Start the status server.
-        if let Err(e) = status_server.start(server_cfg.status_addr) {
+        if let Err(e) = status_server.start(cfg.clone()) {
```

Comment:
Do we need to clone the entire configuration?

Reply:
Good point, it shouldn't be necessary. I will fix it.